tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -587,7 +612,8 @@ def _compareBoth(self, x, y, np_func, tf_func, also_compare_variables=False):
     self._compareCpu(x, y, np_func, tf_func, also_compare_variables)
     if x.dtype in (np.float16, np.float32, np.float64):
       if tf_func not in (_FLOORDIV, math_ops.floordiv, math_ops.igamma,
-                         math_ops.igammac, math_ops.zeta, math_ops.polygamma):
+                         math_ops.igammac, math_ops.zeta, math_ops.polygamma,
+                         _MATMUL):
```

Comment:
I added `_MATMUL` to this list because I was getting an error in `testFloatBasic()` due to a lack of precision in `_compareGradientX()` (which calls `assertAllClose()`).  This might be worth investigating, but it does not seem related to this PR.  Here is the output of the error:

    Not equal to tolerance rtol=0.001, atol=0.001
    
    (mismatch 11.111111111111114%)
     x: array([[ 22.      ,  20.035713,  18.07143 ,   0.      ,   0.      ,
              0.      ,   0.      ,   0.      ,   0.      ],
           [ 16.107143,  14.142857,  12.178572,   0.      ,   0.      ,...
     y: array([[ 22.003172,  20.03479 ,  18.074036,   0.      ,   0.      ,
              0.      ,   0.      ,   0.      ,   0.      ],
           [ 16.098022,  14.144897,  12.176513,   0.      ,   0.      ,...